ui/handler.go

```diff
@@ -121,6 +121,94 @@ func (h *internalHandler) handleNewRulesetRequest(w http.ResponseWriter, r *http
 
 }
 
+func (h *internalHandler) handleEditRulesetRequest(w http.ResponseWriter, r *http.Request) {
+	path := strings.TrimPrefix(r.URL.Path, "/rulesets/")
+	if path == "" {
+		writeError(w, r, nil, http.StatusNotFound)
+		return
+	}
+
+	nrr := &newRulesetRequest{}
+
+	err := json.NewDecoder(r.Body).Decode(nrr)
+	if err != nil {
+		writeError(w, r, err, http.StatusInternalServerError)
```

Comment:
This should be a `http.StatusBadRequest` instead as it's the client's fault

Reply:
Done